client/js/views/Home.vue

```diff
@@ -85,6 +58,9 @@
             </v-btn>
           </v-col>
         </v-row>
+        <v-row class="d-flex justify-end mt-8">
+          <create-game-dialog @error="handleError" />
```

Comment:
I went back and forth on whether to make this a "dumb" presentational component that used props and events to manage creating a game from the Home page.

I opted to make the component talk to the store and manage game creation itself to DRY it up. It seems simpler this way to me

Reply:
I think connecting to the store makes the most sense. There's a time and a place for presentational components and logically I think this is easier to understand.